pkg/tarmak/provider/amazon/amazon.go

```diff
@@ -604,7 +608,34 @@ func (a *Amazon) verifyInstanceType(instanceType string, zones []string, svc EC2
 		ProductDescription: aws.String("Linux/UNIX (Amazon VPC)"),
 		InstanceType:       aws.String(instanceType),
 	}
-	response, err := svc.DescribeReservedInstancesOfferings(request)
+
+	expBackoff := backoff.NewExponentialBackOff()
+	expBackoff.MaxElapsedTime = time.Second * 10
+	ctx, cancelRetries := context.WithCancel(context.Background())
+	b := backoff.WithContext(expBackoff, ctx)
+
+	var response *ec2.DescribeReservedInstancesOfferingsOutput
+	describeInstanceOfferingFunc := func() error {
+		var err error
+		response, err = svc.DescribeReservedInstancesOfferings(request)
+		if err == nil {
+			return nil
+		}
+
+		// if err and it's a rate limiting error, try again
+		if strings.Contains(err.Error(), consts.AmazonRateLimitErr) {
```

Comment:
Can we handle the aws error like that: https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/handling-errors.html

`awserr.Code()` should contain your AmazonRateLimitErr